docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -4,50 +4,44 @@
 <titleabbrev>Add data</titleabbrev>
 ++++
 
-To start working with your data in {kib}, use one of the many ingest options,
-available from the home page.
-You can collect data from an app or service,
-or upload a file that contains your data.  If you're not ready to use your own data,
-add a sample data set and give {kib} a test drive.
+The best way to add data is to use one of our many integrations.
+Alternatively, you add a sample data set or upload a file.
 
-[role="screenshot"]
-image::images/addData_home_7.15.0.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
 [float]
 [[add-data-tutorial-kibana]]
-=== Add data
+=== Add an integration
 
-Want to ingest logs, metrics, security, or application data?
-Install and configure a Beats data shipper or other module to periodically collect the data
-and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
+Integrations periodically collect logs, metrics, security,
```

Comment:
This statement makes it sound like the integrations do the work of collecting data and sending it to ES, which isn't true. The Elastic Agent or Beats shipper does that. 

Here are a couple places where we describe integrations (for inspiration to come up with a more accurate intro): 

https://docs.elastic.co/en/integrations
https://www.elastic.co/guide/en/fleet/7.16/integrations.html